tools/buildsys/src/builder.rs

```diff
@@ -700,6 +700,18 @@ fn secrets_args() -> Result<Vec<String>> {
         );
     }
 
+    let ca_bundle_var = "BUILDSYS_CACERTS_BUNDLE_OVERRIDE";
+    let ca_bundle_value =
+        env::var(ca_bundle_var).context(error::EnvironmentSnafu { var: ca_bundle_var })?;
```

Comment:
Since this is consistent with the existing code, I'd prefer we tackle this (and sbkeys) in a separate PR. I've gone ahead and opened up an issue (#237) so we don't lose track of this.